### PR TITLE
{Compute} Clarify that encrypt-format-all is only applicable to Linux VMs

### PR DESCRIPTION
**Pull request by nachoalonsoportillo**

### CODE REVIEW

**Comment by yonzhan** [comment]
VM

File: src/azure-cli/azure/cli/command_modules/vm/_params.py [hunk 1]
```diff
@@ -472,7 +472,7 @@ def load_arguments(self, _):
         c.argument('disk_name', arg_type=name_arg_type, help='The data disk name.')
 
     with self.argument_context('vm encryption enable') as c:
-        c.argument('encrypt_format_all', action='store_true', help='Encrypts-formats data disks instead of encrypting them. Encrypt-formatting is a lot faster than in-place encryption but wipes out the partition getting encrypt-formatted.')
+        c.argument('encrypt_format_all', action='store_true', help='Encrypts-formats data disks instead of encrypting them. Encrypt-formatting is a lot faster than in-place encryption but wipes out the partition getting encrypt-formatted. (Only supported for Linux virtual machines.)')
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/_params.py:475]: > (Only supported for Linux virtual machines.)

Could you please explain the background of this modification?

Comment (nachoalonsoportillo) [reply src/azure-cli/azure/cli/command_modules/vm/_params.py:475]: Only [Linux extension](https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/azure-disk-enc-linux) supports that encryption operation. [Windows extension](https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/azure-disk-enc-windows) doesn't, and if you specify that parameter for a Windows VM it reports the following error https://github.com/MicrosoftDocs/azure-docs-cli/issues/549.

Comment (zhoxing-ms) [reply src/azure-cli/azure/cli/command_modules/vm/_params.py:475]: Got it, thanks~

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**